docs/source/storage.rst

```diff
@@ -1,7 +1,44 @@
 Storage
 =======
 
-There are 2 storage backends implemented at the moment - ElasticSearch and local.
+The following table provides source and sink configuration fields that you can set for your data pipeline
+
++-----------------+----------+--------+
+| Data Source     | Source   | Sink   |
++=================+==========+========+
+| Elasticsearch   | es       | es     |
++-----------------+----------+--------+
+| Local           | local    | local  |
++-----------------+----------+--------+
+| Local Directory | localdir |        |
++-----------------+----------+--------+
+| Console Output  |          | stdout |
++-----------------+----------+--------+
+| Kafka           |          | kafka  |
++-----------------+----------+--------+
+
+
+
+Example
+-------
+To read from Elasticsearch and write predictions to Kafka you can use the following configurations
+
+.. code-block:: shell
+
+   STORAGE_DATASOURCE: “es”
+   STORAGE_DATASINK: “kafka”
+
+.. image:: ../../imgs/storage-example.png
+
+.. DANGER::
+
+   We have deprecated es.source/es.sink and you need to specify "es" instead. See table above.
```

Comment:
Deprecated would suggest *.source/*.sink is still part of the software, in this PR it seems that that it has been removed entirely, since now we get an exception for this. So I'd suggest rewording to mention it's removed entirely, or just take this bit out imo.

Reply:
Fixed